app/views/layouts/application.html.erb

```diff
@@ -17,6 +17,7 @@
     <%= favicon_link_tag 'govuk-apple-touch-icon.png', rel: 'apple-touch-icon', type: 'image/png' %>
 
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <%= javascript_include_tag "cookie_functions" %>
```

Comment:
is there a reason this isn't just included in the asset bundle with the rest of the JS?

Reply:
To ensure that the cookie functions are available to both the `google_analytics.js` and asset bundle. The asset bundle loads in at the end of the page

Reply:
I've implemented the solution that @erbridge suggested about loading the Google Analytics code on `DOMContentLoaded` so I've put `cookie_functions.js` back in the asset bundle